base/build.gradle

```diff
@@ -32,6 +32,7 @@ dependencies {
     implementation Dependencies.coroutinesCore
     implementation Dependencies.androidxAnnotations
 
+    testImplementation platform(Dependencies.junitBom)
```

Comment:
We might not `base` module in future, but that's something to evaluate later. Added this to fix for now.